rfcs/text/design-tokens.md

```diff
@@ -0,0 +1,154 @@
+- Start Date: (fill me in with today's date, 2021-06-14)
+
+
+- [Summary](#summary)
+- [Basic example](#basic-example)
+- [Motivation](#motivation)
+- [Detailed design](#detailed-design)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+    - [Design tokens via styled-components and CSS variables](#design-tokens-via-styled-components-and-css-variables)
+    - [Tag function for generating token](#tag-function-for-generating-token)
+- [Adoption strategy](#adoption-strategy)
+- [How we teach this](#how-we-teach-this)
+
+# Summary
+
+Implementation of the design tokens with theme support and backwards compatibility.
+
+# Basic example
+
+Design tokens will be kept in `@sage/design-tokens` library. It will be updated every time tokens will change. They will
+be imported from this library and used as the object.
+
+```js
+const Container = styled.div`
+  background: ${props.theme.sagedsColorsBase400};
+  color: ${props.theme.sagedsColorsTextWhite};
+`;
+```
+
+Then, themes would be handled by ThemeProvider.
+
+Themes would be created from existing ones by extending and renaming them, so that current *mint* would be named *sbiz*
```

Comment:
Why?

Why is the mint theme not styled from the design tokens rather than creating another theme.

Reply:
Agree, `mint` was supposed to be the replacement for `small`, so if `sbiz` is intended to be the same thing, why does it need to be a new theme vs just using design tokens in the `mint` theme?

Reply:
It needs to be renamed for keeping naming consistent between code and design system documentation

Reply:
Is the Design System open source?

Reply:
Some parts of it - yes. I see carbon library as an open source part of the design system.
As well as component libraries and tools for other platforms (android, iOS) can be open source I think.